.github/workflows/download-typescript.yml

```diff
@@ -0,0 +1,24 @@
+name: Get Diagnostics From TypeScript
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  get_diagnostics:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download Diagnostics
+        run: curl https://raw.githubusercontent.com/microsoft/TypeScript/main/src/compiler/diagnosticMessages.json -o packages/codefixes/src/diagnosticMessages.json
```

Comment:
I don't think it makes sense to copy this file. I think it's better to download it from TS repo because it's updated frequently.

Reply:
But I think we don't need this step at all :)